ci/prow/boskos/set_permissions.sh

```diff
@@ -41,14 +42,23 @@ for group in ${PROJECT_GROUPS[@]}; do
   gcloud projects add-iam-policy-binding ${PROJECT} --member group:${group} --role roles/editor
 done
 
-# Add all service accounts as editors
 for sa in ${PROJECT_SAS[@]}; do
+  # Add all service accounts as editors and storageAdmins
   echo "NOTE: Adding service account ${sa}"
   gcloud projects add-iam-policy-binding ${PROJECT} --member serviceAccount:${sa} --role roles/editor
+  gcloud projects add-iam-policy-binding ${PROJECT} --member serviceAccount:${sa} --role roles/storage.admin
+  # As required by step 6 in https://github.com/google/knative-gcp/tree/master/docs/storage,
+  # grant the GCS service account the permissions to publish to GCP Pub/Sub
+  gcs_service_account="$(curl -s -X GET -H "Authorization: Bearer $(gcloud auth application-default print-access-token)" "https://www.googleapis.com/storage/v1/projects/${PROJECT}/serviceAccount" \
```

Comment:
No need to grep the account, you already know its name (template using project ID). Just issue the GET API request to get it.

Reply:
The one in its name is actually `project number`, see https://cloud.google.com/storage/docs/projects#service-accounts.
I did some search but did not find a good way to get the `project number`, so I would prefer to keep this command to get the GCS service account.

Reply:
As discussed and tested in person, the project number can be fetched through gcloud app describe.

Reply:
Done. Thanks for the reminder!